serverless.yml

```diff
@@ -0,0 +1,59 @@
+service: site-translation-service
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+  stage: ${env:GDL_ENVIRONMENT, 'test'}
+  region: eu-central-1
+  endpointType: private
+  role: arn:aws:iam::${env:GDL_ACCOUNT, 'DEV'}:role/${env:SERVERLESS_IAM_ROLE, 'DEV'}
+  tags:
+    VERSION: "0.1"
+  deploymentBucket:
+    name: ${env:GDL_SERVERLESS_DEPLOYMENT_BUCKET, 'DEV'}
+  resourcePolicy:
+    - Effect: Allow
+      Principal: "*"
+      Action: execute-api:Invoke
+      Resource:
+        - execute-api:/*/*/*
+      Condition:
+        StringEquals:
+          aws:SourceVpce:
+            - "${env:VPC_ENDPOINT, 'DEV'}"
+
+package:
+  individually: true
+  exclude:
+    - "**__tests__/**"
+
+functions:
+  hello:
+    handler: handler.hello
+    events:
+      - http:
+          path: /{language}
+          method: GET
+          request:
+            parameters:
+              paths:
+                language: true
+
+custom:
+  warmup:
+    enabled:
+      - prod
+  serverless-offline:
+    port: 5000
+  vpc:
+    vpcName: ${env:VPC, 'DEV'}
+    subnetNames:
+      - "${env:GDL_ENVIRONMENT, 'DEV'}.private_a.gdl"
+      - "${env:GDL_ENVIRONMENT, 'DEV'}.private_b.gdl"
+    securityGroupNames:
+      - "serverless-security_group"
+
+plugins:
+  - serverless-plugin-warmup
+  - serverless-vpc-discovery
+  - serverless-offline #!OBS: serverless-offline needs to be last in the list
```

Comment:
Her bør du vel slenge på den overvåkings-pluginen, så vi får varsler

Reply:
Ja, den tenkte jeg ikke over. Legger inn configen for den, så burde vi nesten merge pull requesten med det inn før denne.